core-bundle/src/Monolog/ContaoContext.php

```diff
@@ -45,14 +45,17 @@ public function __construct(
      */
     public function __toString(): string
     {
-        return (string) json_encode([
-            'func' => $this->func,
-            'action' => $this->action,
-            'username' => $this->username,
-            'browser' => $this->browser,
-            'uri' => $this->uri,
-            'pageId' => $this->pageId,
-        ]);
+        return (string) json_encode(
+            [
+                'func' => $this->func,
+                'action' => $this->action,
+                'username' => $this->username,
+                'browser' => $this->browser,
+                'uri' => $this->uri,
+                'pageId' => $this->pageId,
+            ],
+            JSON_THROW_ON_ERROR
```

Comment:
```suggestion
            JSON_THROW_ON_ERROR,
```
Not sure about our coding standard, but I think a trailing comma is missing here.

Reply:
No, it is not, because it is a function call and not an array.